apps/src/lib/tools/jsinterpreter/codegen.js

```diff
@@ -1,4 +1,3 @@
-/* global CanvasPixelArray, Uint8ClampedArray */
 import {dropletGlobalConfigBlocks} from '../../../dropletUtils';
```

Comment:
Having extracted so much from codegen.js, can you describe in prose what the purpose of this file is now?  It'd be nice to have a comment at the top of the file describing its use - or deprecating it, if you think it's just been a dumping ground for interpreter utilities.